Wrapping/Generators/Python/CMakeLists.txt

```diff
@@ -3,9 +3,13 @@
 find_package(PythonInterp)
 find_package(PythonLibs)
 
-# require at least python 2.6
-if(PYTHON_VERSION_STRING VERSION_LESS 2.6)
-  message(WARNING "Python version less than 2.6: \"${PYTHON_VERSION_STRING}\".")
+# require at least python 2.7
+if(PYTHON_VERSION_STRING VERSION_LESS 2.7)
```

Comment:
Check order is wrong. If version is 3.3, the warning message will not be displayed because the outer condition checks <=2.7.

Reply:
(On Windows)

Reply:
Oops -- good catch!